corehq/apps/style/static/style/js/main.js

```diff
@@ -1,287 +1,286 @@
-var COMMCAREHQ = {};
-
-COMMCAREHQ.icons = {
-    GRIP:   'icon-resize-vertical icon-blue fa fa-arrows-v',
-    ADD:    'icon-plus icon-blue fa fa-plus',
-    COPY:   'icon-copy icon-blue fa fa-copy',
-    DELETE: 'icon-remove icon-blue fa fa-remove',
-    PAPERCLIP: 'icon-paper-clip fa fa-paperclip'
```

Comment:
There isn't a good way to do it in LESS, because we run LESS only on HQ's styles, so we can't access the fontawesome classes like `fa`. But I'm fine with putting the constants back: https://github.com/dimagi/commcare-hq/pull/17400/commits/c6c545eff52a6a7d8e38375e2f7227bcefb8c39c